templates/settings-admin.php

```diff
@@ -60,7 +60,7 @@
 		<fieldset>
 			<div class="block">
 				<label><?php p($l->t("Refresh Delay")); ?></label>
-				<input id="gk_refresh_delay" type="text" name="refreshDelay"  value="<?php p($_['refreshDelay']);?>">
+				<input id="gk_refresh_delay" type="text" name="refreshDelay"  value="<?php if (!empty(p($_['refreshDelay']);)) { p($_['refreshDelay']); } ?>">
```

Comment:
Remove ; in the condition to avoid error please : if (!empty(p($_['refreshDelay'])))